dcql/src/dcql-query/m-dcql-trusted-authorities.ts

```diff
@@ -0,0 +1,84 @@
+import * as v from 'valibot'
```

Comment:
Will we switch to zod here as well?

Reply:
Maybe at some point, but since it's a self-contained library I don't have a hurry to update